apps/webClient/src/App.tsx

```diff
@@ -9,10 +9,57 @@ import KCollege from "./views/KCollege";
 
 type Props = {};
 
+const Monsterrat = { fontFamily: "Montserrat, sans-serif", fontWeight: 400 };
+const LibreBaskerville = {
+  fontFamily: "Libre Baskerville, sans-serif",
+  fontWeight: 400,
+};
+
 const theme = createMuiTheme({
+  palette: {
+    background: {
+      default: "#4CBFFE",
+    },
+    primary: {
+      light: "#cfaa67",
+      main: "#4CBFFE",
+      dark: "#89682e",
+    },
+    secondary: {
+      light: "#4f4b4c",
+      main: "#373733",
+      dark: "#181516",
```

Comment:
add constants for all of these hex values so they are easy to read:

```const LIGHT_BLUE = #4CBFFE"```

To be honest it may not make sense to override all of these colors because each of our pages will have different background colors.

Reply:
I agree. We don't really need to change these values here.